pkgs/development/python-modules/safe-pysha3/default.nix

```
@@ -0,0 +1,19 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "safe-pysha3";
+  version = "1.0.3";
+  disabled = pythonOlder "3.8";
```

```suggestion
  version = "1.0.3";
  format = "setuptools";
  disabled = pythonOlder "3.8";
```

Set an explicit format here.